web/impact/impact/tests/test_graphql.py

```diff
@@ -5,7 +5,7 @@
 
 from impact.graphql.middleware import NOT_LOGGED_IN_MSG
 from impact.tests.api_test_case import APITestCase
-from impact.tests.factories import ExpertFactory
+from impact.tests.factories import ExpertFactory, StartupMentorRelationshipFactory  # noqa: E501
```

Comment:
MC style has imports on separate lines:
```
from impact.tests.factories import (
    ExpertFactory, 
    StartupMentorRelationshipFactory,
)
```

Note trailing comma on last line, to facilitate addition of new imports. 
Also note that components are sorted alphabetically within each import.